src/java.base/share/classes/jdk/internal/access/JavaLangAccess.java

```diff
@@ -419,18 +419,21 @@ public interface JavaLangAccess {
 
    /**
     * Get the coder for the supplied character.
+    * @since 21
     */
    @PreviewFeature(feature=PreviewFeature.Feature.STRING_TEMPLATES)
```

Comment:
I think you can drop both @since and @PreviewFeature from these methods. This in an internal interface, used for shared secrets, there is a lot of churn in JLA in each release.

Reply:
Removed